oasis-node segfaults when started without datadir

When `oasis-node` is started without specifying a data directory, it segfaults:
```
ts=2019-11-14T08:15:13.461505557Z level=error module=oasis-node caller=node.go:447 msg="data directory not configured"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1273db7]

goroutine 1 [running]:
github.com/oasislabs/oasis-core/go/common/persistent.(*CommonStore).Close(...)
	github.com/oasislabs/oasis-core/go@/common/persistent/persistent.go:30
github.com/oasislabs/oasis-core/go/oasis-node/cmd/node.(*Node).Cleanup(0xc000098a00)
	github.com/oasislabs/oasis-core/go@/oasis-node/cmd/node/node.go:127 +0x37
github.com/oasislabs/oasis-core/go/oasis-node/cmd/node.newNode.func1(0xc000129b7f, 0xc000098a00)
	github.com/oasislabs/oasis-core/go@/oasis-node/cmd/node/node.go:435 +0x67
github.com/oasislabs/oasis-core/go/oasis-node/cmd/node.newNode(0x1649a00, 0x0, 0x199ccc0, 0xc000dd0950)
	github.com/oasislabs/oasis-core/go@/oasis-node/cmd/node/node.go:448 +0x2ab7
github.com/oasislabs/oasis-core/go/oasis-node/cmd/node.NewNode(...)
	github.com/oasislabs/oasis-core/go@/oasis-node/cmd/node/node.go:413
github.com/oasislabs/oasis-core/go/oasis-node/cmd/node.Run(0x2461340, 0x25d97d8, 0x0, 0x0)
	github.com/oasislabs/oasis-core/go@/oasis-node/cmd/node/node.go:75 +0x2a
github.com/spf13/cobra.(*Command).execute(0x2461340, 0xc0000bc1b0, 0x0, 0x0, 0x2461340, 0xc0000bc1b0)
	github.com/spf13/cobra@v0.0.5/command.go:830 +0x2aa
github.com/spf13/cobra.(*Command).ExecuteC(0x2461340, 0x3f, 0x0, 0x0)
	github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v0.0.5/command.go:864
github.com/oasislabs/oasis-core/go/oasis-node/cmd.Execute()
	github.com/oasislabs/oasis-core/go@/oasis-node/cmd/root.go:43 +0x4b
main.main()
	github.com/oasislabs/oasis-core/go@/oasis-node/main.go:9 +0x20
```

--- a/go/oasis-node/cmd/node/node.go
+++ b/go/oasis-node/cmd/node/node.go
@@ -130,7 +130,9 @@ type Node struct {
 // Cleanup cleans up after the node has terminated.
 func (n *Node) Cleanup() {
 	n.svcMgr.Cleanup()
-	n.commonStore.Close()
+	if n.commonStore != nil {
+		n.commonStore.Close()
+	}
 }
 
 // Stop gracefully terminates the node.

